docs/reference/objectOf.md

```diff
@@ -24,7 +24,11 @@ the validated value is an array that has the exact length of `propsValidation`.
 
 #### `strict`
 If `true` then the result *type-validation* also checks that the validated value
-does not have additional own keys beyond the validated ones.
```

Comment:
Can we add a simple usage example in the beginning? Something like "This is my object, this is how I create the validation, this case will pass, this case will fail" ? It will be very useful for future developers